.vscode/settings.json

```diff
@@ -30,5 +30,4 @@
     },
     "clangd.arguments": [
         "--header-insertion=never"
-    ]
 }
```

Comment:
```suggestion
    ]
}
```

Reply:
Thank you for your reminding and suggestion